index.html

```diff
@@ -82,6 +82,21 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="footer">   
+            <img class="some-icons" src="img/some-icons.svg" alt="facebook and instagram icons">
+            <div class="footer-large">
+                <div class="news-letter-signup">
+                    <h4>Nyhetsbrev?</h4>
+                    <div class="news-letter-input-container">
+                        <input class="news-letter-input" type="text" name="mail" id="mail" placeholder="MAILADDRESS                  ">
```

Comment:
There's a lot of space after "MAILADRESS"?

Reply:
Also, should the type be "email" instead of "text" to improve usability on mobile?